docs/configuration/registration/email.rst

```diff
@@ -14,3 +14,19 @@ building the form.
     
     Email should be configured by the Open Forms supplier and also requires
     proper setup of SPF and/or DMARC-records.
+
+File uploads/attachments
+------------------------
+
+Any files uploaded as part of the submission are not added as email attachments.
+Instead, the e-mail contains download links to the individual files. These links require
+users to be authenticated and have the correct permissions:
+
+* users must be staff users
+* users must have permission to read file attachments. The standard group "Behandelaars"
```

Comment:
Can you then also add it to the actual fixture :)

Reply:
that is in there already, or am I missing something? https://github.com/open-formulieren/open-forms/blob/master/src/openforms/fixtures/default_groups.json#L1607